fighters/common/src/function_hooks/mod.rs

```diff
@@ -29,12 +30,15 @@ pub fn install() {
     is_flag::install();
     controls::install();
     momentum_transfer::install();
+    jumps::install();
     //dash_dancing::install();
 
-    // Handles getting rid of the kill zoom
     unsafe {
+        // Handles getting rid of the kill zoom
         const NOP: u32 = 0xD503201Fu32;
         skyline::patching::patch_data(utils::offsets::kill_zoom_regular(), &NOP);
         skyline::patching::patch_data(utils::offsets::kill_zoom_throw(), &NOP);
+        // Changes full hops to calculate vertical velocity identically to short hops
+        skyline::patching::patch_data(0x6d2188, &0x52800015u32);
```

Comment:
This is OK for now but we are going to need to add byte searching later (this specific instruction is hard to byte search due to local function calls around it)